README.md

```diff
@@ -257,7 +251,36 @@ CustomApp.getInitialProps = async (ctx: AppContext) => {
 };
 ```
 
-<br />
+# ⚗️ CLI (experimental)
+
+You can use `unleash [action] [options]` in your `package.json` `scripts` section, or with:
+
+```sh
+npx @unleash/nextjs
+```
+
+CLI will attemp to read NextJS `.env` files. You need to set `UNLEASH_SERVER_API_URL` and `UNLEASH_SERVER_API_TOKEN` (see [environment variables](#environment-variables)). Another way of configuring the source of feature toggles is to pass environment variables directly, like in the example below. Available shorthand variables are: `UNLEASH_URL`, `UNLEASH_TOKEN`, `UNLEASH_APPNAME`.
```

Comment:
I'm still not sure **why** I need to set these environment variables. What do they do? 

And also, what are the shorthand variables for? They're not mentioned in the "environment variables" section. Why not? Are they only for the CLI? 

`UNLEASH_APPNAME` is also different from `UNLEASH_APP_NAME`. Is that intentional or a typo?

```suggestion
For the CLI to work, the following [environment variables](#environment-variables) must be configured with appropriate values:

- `UNLEASH_SERVER_API_URL` 
- `UNLEASH_SERVER_API_TOKEN`

The CLI will attempt to environment values from any `.env` files if they're present. You can also set the variables directly when invoking the interface, as in the [CLI usage example](#example).
```